ccl: add workloadccl to imports

tbg opened the pull request:
I noticed that the `cockroach-short` binary did not allow `workload init
X --data-loader=IMPORT` and tracked it down to this missing import. It's
probably pulled in accidentally in the full binary.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/41673)
<!-- Reviewable:end -->

knz commented:
LGTM but you got an error remaining in workloadccl

tbg committed:
ccl: add workloadccl to imports

I noticed that the `cockroach-short` binary did not allow `workload init
X --data-loader=IMPORT` and tracked it down to this missing import. It's
probably pulled in accidentally in the full binary.

Release note: None

tbg commented:
TFTR! Fixed the import error.

bors r=knz
